null_safety/calculate_lix/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Files and directories used by the Dart SDK
+.dart_tool/
+.packages
+build/
+doc/api/
+
+# Android Studio configuration
+.idea/
+!.idea/runConfigurations/Run_main_with_test_data.xml
+
+# VSCode configuration
+.vscode/
```

Comment:
The SDK path would be written into `.vscode/settings.json` so you might want to exclude that. Generally I try not to exclude any (since it's often the case that I have [settings I want to commit](https://github.com/Dart-Code/Dart-Code/blob/master/.vscode/settings.json)). VS Code doesn't currently support separate files for settings you do want to commit versus those you don't (like paths), though it's a highly requested feature.